test/e2e/testsuites/dynamically_provisioned_volume_snapshot_tester.go

```diff
@@ -44,6 +44,7 @@ type DynamicallyProvisionedVolumeSnapshotTest struct {
 	CSIDriver                      driver.PVTestDriver
 	Pod                            PodDetails
 	ShouldOverwrite                bool
+	IsCrossRegion                  bool
```

Comment:
there is `location                  string` in `suite_test.go`, so this var is not necessary.

Reply:
use `DynamicallyProvisionedVolumeSnapshotCrossRegionTest` and use different location in test case